Source/cmake/OptionsGTK.cmake

```diff
@@ -385,6 +370,20 @@ if (USE_OPENGL_OR_ES)
         SET_AND_EXPOSE_TO_BUILD(USE_GLX TRUE)
     endif ()
 
+    if (ENABLE_WAYLAND_TARGET AND EGL_FOUND)
```

Comment:
What happens if EGL is not found...? Is it possible to wind up here by mistake, e.g. if the EGL pkg-config file is not installed? The build should fail if a build dep is missing, not proceed with the feature disabled.

Reply:
EGL is required for ENABLE_WAYLAND_TARGET and we make that fatal, but it happens after this, so I just added the AND to avoid doing it when configure is going to fail. Maybe we could move the EGL check before this one, but that's out of scope of this patch, I think.